### PR TITLE
Remove stream directive, define default_dicom_listen_port directly

### LINKED ISSUE #119: Nginx configuration prevents default XNAT SCP from starting
### Short Description

Identified through UCL-MIRSG/UCLMedicalImagingEnv#401, the default configuration for the `nginx` role clashes with the default DICOM SCP receiver that XNAT starts on port 8104.

As nginx starts up first, it starts listening on `nginx_upstream_listen_port` (by default `8104` and so preventst XNAT/Tomcat from listening on it. This means that the DICOM SCP receiver is blocked from starting up.

### Evidence/Steps to Reproduce

See UCL-MIRSG/UCLMedicalImagingEnv#401

### Acceptance Criteria/Expected Behaviour

- The default DICOM receiver starts on Tomcat/XNAT
- Connections can be made to it - either by being forwaded by nginx or by being streamed through it

### Details

_No response_

### Resolution

_No response_

### PR DESCRIPTION
**Pull request by HChughtai**
Closes #119 

This is done by removing the `stream` directive from the `nginx` configuration so that the define port is only listened to by `tomcat`.

### CODE REVIEW

File: playbooks/group_vars/xnat.yml [hunk 1]
```diff
@@ -80,19 +80,18 @@ firewalld_work_zone_open_services:
...
 firewalld_public_zone_ports:
-  - "{{ nginx_upstream_listen_port }}"
+  - "{{ default_dicom_listen_port }}"
 
 # mirsg.infrastructure.nginx
 nginx_use_ssl: "{{ ssl.use_ssl }}"
 nginx_server_name: "{{ web_server.host }}"
-nginx_upstream_port: 8104
-nginx_upstream_listen_port: 8104
+default_dicom_listen_port: 8104
```
Comment (p-j-smith) [inline playbooks/group_vars/xnat.yml:94]: perhaps rename to `nginx_default_dicom_listen_port` (as ansible-lint likes it when variables being with the role name)

**Review by p-j-smith — CHANGES_REQUESTED** [review]
looks good, although one small suggestion around the variable name.

Also, the [test failure](https://github.com/UCL-MIRSG/ansible-collection-infra/actions/runs/9940011557/job/27455997553?pr=126#step:2:1282) should be fixed by #125

File: playbooks/group_vars/xnat.yml (hunk 1 above)
Comment (HChughtai) [reply playbooks/group_vars/xnat.yml:94]: Ah, yeah - good point, thanks!

Comment (HChughtai) [reply playbooks/group_vars/xnat.yml]: FYI I've made the change simpler and more generic by retaining the old `nginx_upstream_listen_port` name

Comment (p-j-smith) [reply playbooks/group_vars/xnat.yml]: good idea! Will make things easier

**Review by p-j-smith — APPROVED** [review]
looks good

**Pull request merged**